patches/0005-Add-placeholders-to-manual-pages.patch

```diff
@@ -95,6 +96,47 @@ index 58d9ad9038..38ebdfacab 100644
  .El
  .Sh SEE ALSO
  .Xr strftime 3 ,
+diff --git src/usr.sbin/bgplgd/bgplgd.8 src/usr.sbin/bgplgd/bgplgd.8
+index a6a1a184bf..3ed19d2630 100644
+--- src/usr.sbin/bgplgd/bgplgd.8
++++ src/usr.sbin/bgplgd/bgplgd.8
+@@ -38,11 +38,11 @@ is a simple server that implements a simple web API to query
+ .Pp
+ .Nm
+ opens a socket at
+-.Pa /var/www/run/bgplgd.sock ,
+-owned by www:www,
++.Pa @WWWRUNSTATEDIR@/bgplgd.sock ,
++owned by @WWW_USER@,
```

Comment:
It is not strictly correct. It works on OpenBSD because the user `www` has primary group `www` but the code
does a `getpwnam` lookup for `@WWW_USER@` and uses the uid and gid of that user.
So in the end it just defaults the `-U` option to `@WWW_USER@.
I was hoping that not mentioning the group would imply that the group ownership is the gid of the user.

Reply:
Ah, thank you for the explanation.